frontend/src/components/custom-tools/prompt-card/OutputForIndex.jsx

```diff
@@ -0,0 +1,49 @@
+import PropTypes from "prop-types";
+import { Modal } from "antd";
+import "./PromptCard.css";
+import { uniqueId } from "lodash";
+
+function OutputForIndex({ chuckData, setIsIndexOpen, isIndexOpen }) {
+  const handleClose = () => {
+    setIsIndexOpen(false);
+  };
+
+  const lines = chuckData?.split("\\n"); // Split text into lines and remove any empty lines
+
+  const renderContent = (chuck) => {
+    return !chuck ? (
+      <p>No chucks founds</p>
+    ) : (
+      <>
+        {chuck?.map((line) => (
+          <div key={uniqueId()}>
+            {line}
+            <br />
+          </div>
+        ))}
+      </>
+    );
```

Comment:
@jagadeeswaran-zipstack NIT: Can this be refactored like this?

```
if (!chunk) {
    return <p>No chucks founds</p>;
}

return (
    <>
        {chuck?.map((line) => (
          <div key={uniqueId()}>
            {line}
            <br />
          </div>
        ))}
      </>
)
```

Reply:
Yes, its more readable.